test/onnx/test_onnx_export.py

```diff
@@ -60,3 +61,47 @@ def break_is_registered_op_api(opname, domain, version):
                               operator_export_type=OperatorExportTypes.ONNX_ATEN_FALLBACK)
         onnx_model = onnx.load_from_string(f.getvalue())
         self.assertAtenOp(onnx_model, "clamp", "Tensor")
+
+
+    def _helper_test_to_(self, cast_fn):
+        """Run a generic test with the specified cast_fn
+
+        `cast_fn` will be converted into a `torch.jit.script` function
+        and it is meant to wrap a `aten::to` during export to prevent
+        hard-coded devices
+
+        `cast_fn` signature is:
+            def cast_fn(input: torch.Tensor) -> torch.Tensor
+        """
+        cast_fn = torch.jit.script_if_tracing(cast_fn)
+
+        class MyModel(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv1 = torch.nn.Conv2d(3, 20, 5)
+                self.conv2 = torch.nn.Conv2d(20, 20, 5)
+
+            def forward(self, x):
+                x = cast_fn(x)
+                x = torch.nn.functional.relu(self.conv1(x))
+                return torch.nn.functional.relu(self.conv2(x))
```

Comment:
is the double conv and double relu needed? If not, can we simplify the model so it's clear what is actually being tested?